src/test/java/ancaMarian/tests/tema5/DemoQA.java

```diff
@@ -0,0 +1,215 @@
+package ancaMarian.tests.tema5;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class DemoQA {
+    private static String url;
+    private static ChromeDriver driver = null;
+
+    DemoQA(String url, ChromeDriver driver) {
+        this.url = url;
+        this.driver = driver;
+        driver.get(url);
+    }
+    public static void checkTextBox() {
```

Comment:
You're missing here a blank line between constructor an this method. Please take into account to make this change too on next homework. Thanks!